To: vim-dev@vim.org
Subject: Patch 7.3.002
Fcc: outbox
From: Bram Moolenaar <Bram@moolenaar.net>
Mime-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit
------------

Patch 7.3.002
Problem:    ":find" completion doesn't work when halfway an environment
	    variable. (Dominique Pelle)
Solution:   Only use in-path completion when expanding file names. (Nazri
	    Ramliy)
Files:	    src/ex_docmd.c


*** ../vim-7.3.001/src/ex_docmd.c	2010-08-15 21:57:27.000000000 +0200
--- src/ex_docmd.c	2010-08-16 20:51:22.000000000 +0200
***************
*** 3465,3471 ****
  	case CMD_find:
  	case CMD_sfind:
  	case CMD_tabfind:
! 	    xp->xp_context = EXPAND_FILES_IN_PATH;
  	    break;
  	case CMD_cd:
  	case CMD_chdir:
--- 3465,3472 ----
  	case CMD_find:
  	case CMD_sfind:
  	case CMD_tabfind:
! 	    if (xp->xp_context == EXPAND_FILES)
! 		xp->xp_context = EXPAND_FILES_IN_PATH;
  	    break;
  	case CMD_cd:
  	case CMD_chdir:
*** ../vim-7.3.001/src/version.c	2010-08-16 21:46:12.000000000 +0200
--- src/version.c	2010-08-16 22:33:13.000000000 +0200
***************
*** 716,717 ****
--- 716,719 ----
  {   /* Add new patch number below this line */
+ /**/
+     2,
  /**/

-- 
Warning label on a superhero Halloween costume:
"Caution: Cape does not enable user to fly."

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\        download, build and distribute -- http://www.A-A-P.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///
